spec.html

```diff
@@ -29663,7 +29663,7 @@ <h1>EvalError ( . . . )</h1>
       <p>See <emu-xref href="#sec-native-error-types-used-in-this-standard-evalerror"></emu-xref>.</p>
     </emu-clause>
 
-    <emu-clause id="sec-constructor-properties-of-the-global-object-finnalization-registry">
+    <emu-clause id="sec-constructor-properties-of-the-global-object-finalization-registry">
```

Comment:
Not sure if it's worth adding `oldids` for this thb

Reply:
I'd rather just keep the ID the way it is. But if we change it, yes, we should use an oldid.

Reply:
I doubt anyone's referenced the typoed URL in the last 24 hours - we should definitely change it, and I don't think it matters if there's an oldid or not.

Reply:
This isn't a new clause.

Reply:
@linusg Let's at least do it in a separate PR please.

Reply:
ah, then yes it definitely needs an old ID

Reply:
-> https://github.com/tc39/ecma262/pull/3458``